lib/passport-saml/strategy.js

```diff
@@ -20,6 +20,12 @@ function Strategy (options, verify) {
   this._saml = new saml.SAML(options);
   this._passReqToCallback = !!options.passReqToCallback;
   this._authnRequestBinding = options.authnRequestBinding || 'HTTP-Redirect';
+
+  // option to use a custom POST form for AuthnRequests.
+  // Provide a function which takes an array of { name: , value: } objects.
+  // The values have already been escaped.
+  // Return the form as a string.
+  this._customAuthnRequestForm = options.authnRequestPostForm;
```

Comment:
Suggest changing the naming for consistency: `this._ authnRequestPostForm = options.authnRequestPostForm`